File: 151001/Beryozkin/REST/src/main/java/app/entities/Author.java
package app.entities;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class Author {
    private Long id;
    private String login;
    private String password;
    private String firstname;
    private String lastname;
}

File: 151004/Afanasenko/appLab4/common/src/main/kotlin/com/github/kornet_by/dc/lab4/bean/Author.kt
package com.github.kornet_by.dc.lab4.bean

import com.github.kornet_by.dc.lab4.dto.response.AuthorResponseTo
import kotlinx.serialization.Serializable

@Serializable
data class Author(
	val id: Long?, val login: String, val password: String, val firstname: String, val lastname: String
) {
	fun toResponse(): AuthorResponseTo = AuthorResponseTo(id, login, password, firstname, lastname)
}